### PR TITLE
New items, tweaks

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request

-several interesting items can now appear in surplus crates
-viro can now buy several different disease cultures. Some, like Spanish Flu and Pierrot's throat, are available to their respective roles.
-Clown can now get clown spawnergrenades, and clownbomb is cheap enough to be worth it
-an insane clown or chaplain can get a superior honkrender for all 20 TC. a clown can get a honkrender for the same price as a clown bomb. Spawns less clowns but is far easier to deploy
-cook can get gluttony's blessing and a veal render for... whatever reason they need them

## Why It's Good For The Game

Adds some fun new items to the game. Viro can buy some fun cultures instead of being incompetent or just flat out killing everyone, clown can get really clowny now, and more!
## Changelog
:cl:
add: Added several new role specific items
tweak: Several new items can appear in surplus crates, albiet rarely
balance: Lowered cost on clown bomb
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
when you say surplus do you mean nullcrates

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -1649,6 +1683,62 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+			At least, that's what the label says."
+	item = /obj/item/reagent_containers/glass/bottle/pierrot_throat
+	cost = 5
+	restricted_roles = list("Virologist, Clown")
+	
+/datum/uplink_item/role_restricted/random_disease
+	name = "Experimental Disease"
+	desc = "A random disease. Maybe you'll get lucky."
+	item = /obj/item/reagent_containers/glass/bottle/random_virus
+	cost = 1
```
Comment (qwertyquerty) [inline code/modules/uplink/uplink_items.dm:1726]: Couldn't they just buy a bunch of these and likely get a really good one

**Comment by qwertyquerty** [comment]
Review your prices. I think some of these are too low price or to high of chance for surplus

File: code/modules/uplink/uplink_items.dm (hunk 1 above)
Comment (zeskorion) [reply code/modules/uplink/uplink_items.dm:1726]: No. Higher symptoms are rare to spawn and optimal diseases are rarer

**Comment by zeskorion** [comment]
> Review your prices. I think some of these are too low price or to high of chance for surplus

Alright, changed up a few prices. Surplus chances for the big ones, however, are only twice as high as his grace, which is quite low.

> you also shouldnt make miner items available for buying with TC like veal render and glutons blessing, it defeats the point of their lavaland ruins existing

yeah, removin those two

**Comment by zeskorion** [comment]
I also made it so the new surplus additions will NOT appear in surplus. Additionally, romerol is now far rarer in surplus

**Comment by ghost** [comment]
Surplus isn't a variable Zesk.

**Comment by qwertyquerty** [comment]
What are these ported from

**Comment by Toxici11i** [comment]
qwerty all of these are already in code, maybe except the honkrender one but thats just modified

**Comment by Toxici11i** [comment]
they were never added to the uplink though lmao

**Comment by zeskorion** [comment]
> What are these ported from

what toxic said. those like the CLUWNE grenades look like they may've been intended for the uplink but just never added.

File: code/modules/uplink/uplink_items.dm [hunk 2]
```diff
@@ -1606,9 +1608,27 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			transported to you that will teleport the actual bomb to it upon activation. Note that this bomb can \
 			be defused, and some crew may attempt to do so."
 	item = /obj/item/sbeacondrop/clownbomb
-	cost = 15
+	cost = 8
```
Comment (qwertyquerty) [inline code/modules/uplink/uplink_items.dm]: This seems super cheap

File: code/modules/uplink/uplink_items.dm [hunk 3]
```diff
@@ -576,6 +578,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			along with slurred speech, aggression, and the ability to infect others with this agent."
 	item = /obj/item/storage/box/syndie_kit/romerol
 	cost = 28
+	surplus = 10
```
Comment (qwertyquerty) [inline code/modules/uplink/uplink_items.dm]: This seems too high considering the price

File: code/modules/uplink/uplink_items.dm [hunk 4]
```diff
@@ -537,7 +538,8 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			deflecting all ranged weapon fire, but you also refuse to use dishonorable ranged weaponry."
 	item = /obj/item/book/granter/martial/carp
 	cost = 16
-	surplus = 0
+	surplus = 10
```
Comment (qwertyquerty) [inline code/modules/uplink/uplink_items.dm:541]: Too high in my opinion

File: code/modules/uplink/uplink_items.dm [hunk 5]
```diff
@@ -401,7 +401,8 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			organic host as a home base and source of fuel. Holoparasites come in various types and share damage with their host."
 	item = /obj/item/storage/box/syndie_kit/guardian
 	cost = 18
-	surplus = 0
+	surplus = 10
```
Comment (qwertyquerty) [inline code/modules/uplink/uplink_items.dm:404]: Way too high in my opinion

File: code/modules/uplink/uplink_items.dm (hunk 3 above)
Comment (zeskorion) [reply code/modules/uplink/uplink_items.dm]: Note that these are not percentage chances of appearing. I think they're the percentage chance it is included in the pool at all. It used to be ALWAYS included

File: code/modules/uplink/uplink_items.dm (hunk 4 above)
Comment (zeskorion) [reply code/modules/uplink/uplink_items.dm:541]: seems high in paper but testing shows it's extremely rare. Especially considering how much it costs for a surplus crate and how infrequently they are picked

File: code/modules/uplink/uplink_items.dm (hunk 5 above)
Comment (zeskorion) [reply code/modules/uplink/uplink_items.dm:404]: ditto

File: code/modules/uplink/uplink_items.dm (hunk 4 above)
Comment (qwertyquerty) [reply code/modules/uplink/uplink_items.dm:541]: We don't want surplus crates to become the new meta either.

**Pull request merged**